transformations/sentiment_emoji_augmenter/transformation.py

```diff
@@ -0,0 +1,115 @@
+import random
+
+from interfaces.SentenceOperation import SentenceAndTargetOperation
+import spacy
+from tasks.TaskTypes import TaskType
+
+"""
+Adds a positive labelled emoji as well as a positive emoteicon for positive sentences and vice versa. 
+And neutral smiley for unlabelled and neutral sentences.
+Since IMDB has labels +1 --> str(target) in ["1", "pos", "positive"] is used to determine if it's positive.
+"""
+
+emoji = {  # (facial expression, sentiment)-keys
```

Comment:
Same as below - the thresholds seems somewhat arbitrary. I don't understand why, e.g.,  😎 wouldn't be used by a very positive review.